src/server/actions.ts

```diff
@@ -77,6 +78,19 @@ export default class Actions {
     try {
       if (!this.account) throw new Error('account not ready');
 
+      const { data } = await apiInstance.query.system.account(address);
```

Comment:
Maybe better to move this to a separate action and use before sending, after checking `isPrivileged` for example. Because parity and web3 members shouldn't have that restriction IMO.